pennylane/plugins/tests/conftest.py

```diff
@@ -79,6 +79,40 @@ def _skip_if(dev, capabilities):
     return _skip_if
 
 
+@pytest.fixture(scope="session")

```

Comment:
I never got comfortable with the `scope`. What does it mean in this setting to set the scope as `"session"`?